cli/export.go

```diff
@@ -188,5 +208,9 @@ func printEnv(input ExportCommandInput, credsProvider aws.CredentialsProvider, p
 	if creds.CanExpire {
 		fmt.Printf("%sAWS_CREDENTIAL_EXPIRATION=%s\n", prefix, iso8601.Format(creds.Expires))
 	}
+	if region != "" {
+		fmt.Printf("%sAWS_REGION=%s\n", prefix, region)
```

Comment:
Could we also set AWS_DEFAULT_REGION too in order to support [boto3 python clients](https://docs.aws.amazon.com/sdkref/latest/guide/feature-region.html)?